src/sass/components/footer.scss

```diff
@@ -0,0 +1,126 @@
+.footer {
+	font-size: 14px;
+	border-top: 0;
+	padding: uikit-space( 1.75 ) 0 uikit-space( 1 ) 0;
+
+	&.footer--dark {
+		background-color: $uikit-colour-Secondary;
+		color: #fff;
+
+		a {
+			color: $uikit-colour-Text-30;
+		}
+	}
+
+	&.footer--blue {
+		background-color: $uikit-colour-Primary;
+		color: #fff;
+
+		a {
+			color: #fff;
+		}
+	}
+
+	&.footer--light {
+		background-color: #fff;
+		color: $uikit-colour-Text;
+
+		a {
+			color: $uikit-colour-Text;
+		}
+	}
+}
+
+.footer__logo {
+	width: 118px;
+	max-width: 100%;
+	margin-bottom: uikit-space( 1.3 );
+
+	@include uikit-media( md ) {
+		margin-bottom: 0;
+	}
+}
+
+.uikit-body {
+
+	&.footer--blue {
+
+		.footer__links__wrapper {
+			border-bottom: 1px solid $uikit-colour-Primary-60;
+			border-top: 1px solid $uikit-colour-Primary-60;
+		}
+
+		.uikit-link-list--inverted {
+			background-color: $uikit-colour-Primary;
+		}
+	}
+
+	&.footer--light {
+
+		.footer__links__wrapper {
+			border-bottom: 1px solid $uikit-colour-Border;
+			border-top: 1px solid $uikit-colour-Border;
+		}
+
+		.uikit-link-list--inverted {
+			background-color: #fff;
+			color: $uikit-colour-Text;
+
+			a {
+				color: $uikit-colour-Text;
+			}
+		}
+	}
+
+	.footer__links__wrapper {
+		border-bottom: 1px solid $uikit-colour-Border;
+		border-top: 1px solid $uikit-colour-Border;
+		padding: uikit-space( 0.35 ) 0 0 0;
+		margin-bottom: uikit-space( 0.75 );
+
+		@include uikit-media( md ) {
+			border-top: 0 solid $uikit-colour-Border;
+		}
+	}
+
+	.footer__listitem {
+		> .uikit-link-list.uikit-link-list--inline & {
+			margin: 0 uikit-space( 0.5 ) uikit-space( 1 ) uikit-space( 0.5 );
+			display: block;
+
+			@include uikit-media( sm ) {
+				width: auto;
+				display: inline-block;
+			}
+		}
+	}
+
+	.footer__links {
+		display: block;
+		margin-right: uikit-space( -0.5 );
+		margin-left: uikit-space( -0.5 );
+		padding: uikit-space( 0.5 ) 0 uikit-space( 0.25 ) 0;
+	}
+
+	.footer__link {
+		padding-bottom: uikit-space( 0.125 );
+	}
+
+	.footer__text--small {
+		font-size: 87.5%;
+		-webkit-font-smoothing: antialiased;
```

Comment:
nice one. autoprefixer should be doing that but I know it misses sometimes. Let's check here